codeserver/ubi9-python-3.9/run-code-server.sh

```diff
@@ -16,6 +16,31 @@ fi
 # Initilize access logs for culling
 echo '[{"id":"code-server","name":"code-server","last_activity":"'$(date -Iseconds)'","execution_state":"running","connections":1}]' > /var/log/nginx/vscode.access.log
 
+# Check if code-server exists
+if [ ! -f "/opt/app-root/src/.local/share/code-server" ]; then
+
+    # Check internet connection
+    if curl -Is http://www.google.com | head -n 1 | grep -q "200 OK"; then
+        # Internet connection is available
+        echo "Internet connection available. Installing specific extensions."
+
+        # Install specific extensions
+        code-server --install-extension ${SCRIPT_DIR}/utils/ms-python.python-2023.14.0.vsix
+        code-server --install-extension ${SCRIPT_DIR}/utils/ms-toolsai.jupyter-2023.3.100.vsix
+    else
+        # No internet connection
+        echo "No internet connection. Installing all extensions."
+
+        # Install all extensions
+        code-server --install-extension ${SCRIPT_DIR}/utils/ms-python.python-2023.14.0.vsix
```

Comment:
would `code-server --install-extension ${SCRIPT_DIR}/utils/*.vsix` also work?
For those users that modify the default set.
I am also thinking: would it be possible to put the list of binary extensions in a snippets folder location?

Reply:
Thanks for this suggestion, we can make it general.

Reply:
good to know, thank you.